docs/spec/auth/vesting.md

```diff
@@ -1,159 +1,138 @@
-## Vesting
+# Vesting
 
-### Intro and Requirements
+## Intro and Requirements
 
-This paper specifies vesting account implementation for the Cosmos Hub. 
-The requirements for this vesting account is that it should be initialized during genesis with
-a starting balance X coins and a vesting endtime T. The owner of this account should be able to delegate to validators 
-and vote with locked coins, however they cannot send locked coins to other accounts until those coins have been unlocked. 
-The vesting account should also be able to spend any coins it receives from other users. 
-Thus, the bank module's `MsgSend` handler should error if a vesting account is trying to send an amount that exceeds their 
-unlocked coin amount.
+This paper specifies vesting account implementation for the Cosmos Hub.
+The requirements for this vesting account is that it should be initialized
+during genesis with a starting balance `X` coins and a vesting end time `T`.
 
-### Implementation
+The owner of this account should be able to delegate to validators
+and vote with locked coins, however they cannot send locked coins to other
```

Comment:
Can they deposit to proposals? Not sure if we've decided that yet.

Reply:
No we have not decided on that as far as I know. I'll make a note of it in the spec as something to consider in the future.

Reply:
Makes sense, let's note it (cc @sunnya97 thoughts)?

Reply:
Tabled to later per discussion in standup call.